src/main/java/com/faforever/client/api/FafApiAccessorImpl.java

```diff
@@ -326,9 +327,9 @@ public Mod getMod(String uid) {
 
   @Override
   public Optional<MapVersion> findMapByFolderName(String folderName) {
-    List<MapVersion> maps = getMany(MAP_ENDPOINT, 1, ImmutableMap.of(
-        "include", "latestVersion,author",
-        "sort", "-updateTime"));
+    List<MapVersion> maps = getMany("/data/mapVersion", 1, ImmutableMap.of(
+        "filter", MessageFormat.format("filename==\"*{0}*\"", folderName),
```

Comment:
`MessageFormat` is for human-readable strings and it adds locale-specific things like separators in `1'000'000` vs `1,000,000` and such. It's, therefore, not suitable for technical string building; use the "dumb" `String.format()` instead.